tests/common.rs

```diff
@@ -78,6 +78,14 @@ pub fn check_result(res: &VmResult) {
 	}
 }
 
+#[allow(dead_code)]
+pub fn get_fixture_path() -> PathBuf {
+	let mut fixture_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+	fixture_path.push("tests/data/fixtures/fs");
```

Comment:
if this fn only returns the _fs_ fixtures, it should be named `get_fs_fixture_path`.